### ISSUE TITLE
Styling in latest update affect width of dropdownlists

### ISSUE THREAD

**Issue by bjarnef** [DanDiplo/UmbracoTraceLogViewer#23]
I have a site running on UaaS / Umbraco Cloud and it has automatically updated from 7.5.3 to 7.5.4 a few dates ago.
In the commits there are updates to the log trace viewer.

It seems these lines affect width of dropdownlists, e.g. in datatypes:
https://github.com/DanDiplo/UmbracoTraceLogViewer/blob/master/Umbraco7.1.3/App_Plugins/DiploTraceLogViewer/backoffice/diplotracelog/tracelog2.css#L43-L49

![image](https://cloud.githubusercontent.com/assets/2919859/19724859/dab27e5c-9b82-11e6-96e1-9b24fae2c86e.png)

![image](https://cloud.githubusercontent.com/assets/2919859/19724840/c0ecd2d8-9b82-11e6-932b-606f3d26ea85.png)

This should fix it if you change it to the following:
```
#traceLog tr.filters input, #traceLog tr.filters select {
    width: 100%;
    margin: 0;
    padding: 5px;
    height: 30px;
    box-sizing: border-box;
}
```

**Comment by DanDiplo** [comment]
Good spot, I'd noticed dropdowns had gone like that, but thought it was an Umbraco change! Silly me. Your fix looks correct - I'll add it to the next release. Sorry for any inconvenience.

**Comment by DanDiplo** [comment]
I've released a new version (2.3.1) with this CSS fix. It should be on NuGet by the time you read this:

https://www.nuget.org/packages/DiploTraceLogViewer/2.3.1

I'll close this for now, but feel free to re-open if you still see issues. Thanks for reporting.
